### ISSUE TITLE
DashCam: Scoreboard should be sorted on score

### ISSUE THREAD

**Issue by GeleArthur** [GeleArthur/DashCam#20]

**Comment by styledev** [comment]
The current HD Hud doesn’t resort based on score and I am wondering if this will get to distracting. Or is this something else?

**Comment by zedrix** [comment]
Yes, you're right, the spectator hud doesn't sort but it doesn't show score either, so there's no need to sort. on the contrary the DashCam Hud should sort by score because it also shows the score. I see it as something between the spectator hud and the game menu where the scoreboard is sorted by score.
Sorting helps the viewer see the score faster without wasting a lot of time finding the top player on the team and being distracted from the live match.

**Comment by GeleArthur** [comment]
We do show the player score? It switches one and off.

**Comment by GeleArthur** [comment]
#20
